utilities/test_suite/HOST/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_suite/HOST)
+
+set(ROCM_PATH /opt/rocm CACHE PATH "default ROCm installation path")
+
+# Set message options
+if(NOT WIN32)
+    string(ASCII 27 Esc)
+    set(ColourReset "${Esc}[m")
+    set(Red         "${Esc}[31m")
+    set(Green       "${Esc}[32m")
+    set(Yellow      "${Esc}[33m")
+    set(Blue        "${Esc}[34m")
+    set(BoldBlue    "${Esc}[1;34m")
+    set(Magenta     "${Esc}[35m")
+    set(Cyan        "${Esc}[36m")
+    set(White       "${Esc}[37m")
+endif()
+
+find_package(OpenCV QUIET)
+find_package(TurboJpeg QUIET)
```

Comment:
@sampath1117 Please add and verify TurboJPEG install instructions with any specific version if needed. (Add it in RPP's external readme under the Test Suite PreReqs - https://github.com/sampath1117/rpp#prerequisites-for-test-suite)